www/loginuserpass.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-use Sil\Psr3Adapters\Psr3SamlLogger;
+use Sil\Psr3Adapters\Psr3StdOutLogger;
```

Comment:
Why is `Psr3StdOutLogger` used here, but `Psr3EchoLogger` is used elsewhere?

Reply:
For production code, Psr3StdOutLogger is used, as it bypasses PHP's output buffering.

In tests, we want the logs to be included in the buffered output (so that error messages show up in the relevant place within the test's output), so Psr3EchoLogger is used, which _is_ affected by PHP's output buffering.